sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslSqlStdOperatorsTest.java

```diff
@@ -1047,7 +1046,9 @@ public void testTruncate() {
   @SqlOperatorTest(name = "RAND", kind = "OTHER_FUNCTION")
   public void testRand() {
     ExpressionChecker checker =
-        new ExpressionChecker().addExpr("RAND(c_integer)", new Random(INTEGER_VALUE).nextDouble());
+        new ExpressionChecker()
+            .addExpr(
+                "RAND(c_integer)", new Random(INTEGER_VALUE ^ (INTEGER_VALUE << 16)).nextDouble());
```

Comment:
So Beam SQL rand is always initialized with the same seed? Is this what we want?

Reply:
This is actually testing Calcite now. I'm guessing this behavior is determined by the SQL standard, but we probably don't do the right thing if there is more than one thread/worker.